features/configuration/fail_fast.feature

```diff
@@ -1,19 +1,13 @@
 Feature: fail fast
 
-  Use the `fail_fast` option to tell RSpec to abort the run on first failure:
+  Use the `fail_fast` option to tell RSpec to abort the run on after N failures:
 
-  ```ruby
-  RSpec.configure { |c| c.fail_fast = true }
-  ```
```

Comment:
This is documentation and should be restored (although you can add a comment that true means 1)